pkgs/modules/nodejs/default.nix

```
@@ -15,6 +15,8 @@ let
     ${prybar}/bin/prybar-nodejs -q --ps1 "''$(printf '\u0001\u001b[33m\u0002\u0001\u001b[00m\u0002 ')" -i ''$1
   '';
 
+  prettier = pkgs.nodePackages.prettier;
```

Usually I prefer to pin the node.js used by the tools to the same version as supplied by the module. Using `nodepkgs` instead of `pkgs.nodePackages` does this.
```suggestion
  prettier = nodepkgs.prettier;
```